scm/test_helper.go

```diff
@@ -2,24 +2,30 @@ package scm
 
 import (
 	"context"
+	"testing"
 
 	"github.com/beatlabs/github-auth/app"
+	"github.com/quickfeed/quickfeed/internal/env"
 )
 
-func TestSCMManager() *Manager {
+// TestSCMManager sets the current provider to "fake", creates a "test" organization
+// and a fake scm client for this organization.
+func TestSCMManager(t *testing.T) (SCM, *Manager) {
+	t.Helper()
+	env.SetFakeProvider(t)
 	conf := &Config{
 		"test",
 		"test",
 		&app.Config{},
 	}
 	sc := NewFakeSCMClient()
 	_, _ = sc.CreateOrganization(context.Background(), &OrganizationOptions{
```

Comment:
Since `CreateOrganization` is only used by the fake (mock) provider, maybe we could simply remove it and instead just return a "mocked" organization via the `GetOrganization` that may be initialized via NewFakeSCM if necessary.

If you agree, then we can fix this as a separate PR, but would be good to create an issue for it first that can be closed after together with the PR.

Reply:
Agree. I am working on the mock scm client right now, and this adjustment looks like a good match for the same PR. Will create a related issue.

Reply:
GitLab scm client also uses the same method, is it safe to remove?

Reply:
Yes. I think we should remove it. If we add support for GitLab in the future, perhaps their APIs have changed so much that we have to do other things anyway, e.g., maybe they also have support for Apps now?